src/csharp/Microsoft.Spark/Sql/Column.cs

```diff
@@ -743,6 +743,80 @@ public Column Over()
             return Apply("over");
         }
 
+
+        /// <summary>
+        ///  A boolean expression that is evaluated to true if the value of this expression 
+        ///  is contained by the evaluated values of the arguments.  
+        /// </summary>
+        /// <returns>Column object</returns>
+        public Column IsIn(IEnumerable<string> list){
```

Comment:
The Scala version takes in varargs, so we should do something similar with `params`.